package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cognite/react-timeseries-chart",
-  "version": "0.2.3",
+  "version": "0.3.0",
```

Comment:
Since this is a breaking change (changing prop names and expected prop values), should we do a major release?

Reply:
Yes great idea

Reply:
Its 1.0.0-alpha now